sabre1041/managing-ocp-install-beyond
# Cleanup instances

Playbooks to clean up OCP or All instances (including Tower) and Tower configuration for each.

This PR also adds descriptions to each object for clarity.

To test, set these variables:

```
tower_config: true
tower_workflow_job_deploy_launch: true
tower_workflow_job_scaleup_launch: true
tower_project_provision_and_configure_url: "https://github.com/vvaldez/summit-2017-ocp-operator.git"
tower_project_provision_and_configure_branch: "cleanup-instances"
```

Now, go into Tower and verify the cluster deployed. 

Lastly, launch the Workflow Job "3-Terminate_All".

Alternatively, you can also set:
```
tower_workflow_job_terminate_launch: true

```

This will build the cluster, add a node, then tear it down.
